### PR TITLE
Jakarta Authorization 1.5 apidocs

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: arjantijms <arjan.tijms@gmail.com>

- [X] Specification JavaDoc in the wombat/x.y/apidocs directory.

### CODE REVIEW

File: authorization/1.6/apidocs/allclasses-index.html [hunk 1]
```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE HTML>
+<!-- NewPage -->
+<html lang="en">
+<head>
+<!-- Generated by javadoc (12.0.1) on Sun Jul 14 22:30:54 CEST 2019 -->
```
Comment (ggam) [inline authorization/1.6/apidocs/allclasses-index.html]: Minor issue, but some JavaDocs are generated with JDK 8, others with JDK 11 or 12. Won't this generate inconsistent stylings?

Btw: I particularly like JDK 11+ JavaDoc as it contains the search funcionality

Comment (bshannon) [reply authorization/1.6/apidocs/allclasses-index.html]: @ggam The combined platform javadocs are regenerated from source.

Comment (arjantijms) [reply authorization/1.6/apidocs/allclasses-index.html]: Good remark though @ggam, I was actually noticing that as well, but indeed the combined docs are regenerated.

There's a couple of other small inconsistencies, like the pdf files for the spec sometimes have -spec in the file name and sometimes not.

**Comment by bshannon** [comment]
The apidocs look fine, marking this final.

**Pull request merged**